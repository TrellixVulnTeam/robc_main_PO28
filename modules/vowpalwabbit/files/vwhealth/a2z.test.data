1 | Adventure
1 | Fun
1 | Dutch_Ovens
1 | NOT_IN_INPUT
1 | Helpful
1 | Outdoors
1 | Values
