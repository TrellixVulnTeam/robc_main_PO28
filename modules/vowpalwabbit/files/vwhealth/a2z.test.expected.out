0.975723
	Constant:116060:1:0.545758	 ^Adventure:24566:1:0.429965
0.522450
	Constant:116060:1:0.545758	 ^Fun:52006:1:-0.0233073
0.602105
	Constant:116060:1:0.545758	 ^Dutch_Ovens:21943:1:0.0563475
0.545758
	Constant:116060:1:0.545758	 ^NOT_IN_INPUT:139908:1:0
0.361525
	Constant:116060:1:0.545758	 ^Helpful:76812:1:-0.184232
0.942800
	Constant:116060:1:0.545758	 ^Outdoors:234001:1:0.397043
0.398679
	Constant:116060:1:0.545758	 ^Values:63861:1:-0.147078
